source/nodejs/marked-schema/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
@matthidinger - do we have any concerns with just bringing this package in with its existing license?